types/karma/index.d.ts

```diff
@@ -700,6 +692,31 @@ export interface BrowserConsoleLogOptions {
     terminal?: boolean;
 }
 
+interface ParseOptions {
+    /**
+     * When true, the return value of the function is a Promise of Config instead of Config.
+     * Should be set to true to support async Karma configuration file.
+     *
+     * @deprecated Will become a default in the next major release.
+     */
+    promiseConfig: boolean;
+
+    /**
+     * When true, function will throw error or return rejected Promise instead of calling process.exit(1).
+     *
+     * @deprecated Will become a default in the next major release.
+     */
+    throwErrors: boolean;
```

Comment:
```suggestion
    throwErrors?: boolean;
```
should this be optional? I cannot see sample snip using this with `false` (contrary to `promiseConfig`), always as `true` or comparing against truthy/true (=== or if(...))